Riot/Modules/Room/RoomViewController.m

```diff
@@ -276,10 +276,23 @@ + (NSString *)className
     return result;
 }
 
+/// Register provider for Pills.
++ (void)registerPillAttachmentViewProviderIfNeeded
```

Comment:
Feels a bit weird to have this on the same level as the constructors. Seems more like a private method to me.